shared/chat/inbox/container.js

```diff
@@ -46,26 +46,26 @@ const passesParticipantFilter = (filter: string, participants: Array<string>, yo
 // Simple score for a filter. returns 1 for exact match. 0.75 for full name match
 // in a group conversation. 0.5 for a partial match
 // 0 for no match
-function scoreFilter(filter, stringToFilterOn, you) {
-  if (stringToFilterOn.indexOf(',') > 0) {
-    const participants = stringToFilterOn.split(',')
-    if (participants.some(p => p.toLowerCase() === filter.toLowerCase())) {
+function scoreFilter(filter: string, stringToFilterOn: string, participants: Array<string>, you: string) {
+  const lcFilter = filter.toLowerCase()
```

Comment:
i mean filter in getFilter could always lowercase so we do it once really instead of per call of this function

Reply:
sure, but you want this function to work even if it isn't called with a lowercase version of filter. Otherwise it's implicitly dependent on the right input with no constraint.